apache/application.wsgi.mako

```diff
@@ -0,0 +1,21 @@
+import site
+import sys
+import re
+
+site.addsitedir("${python_path}")
```

Comment:
I hope this change will work for the geoportailv3 project. @sbrunner are you confident with it? ;)